client/app/src/js/services/services.js

```diff
@@ -58,7 +58,31 @@ function Auth($http, $location, $window) {
 };
 Auth.$inject = ['$http', '$location', '$window'];
 
+function Screenshots($http) {
+
+  var grabScreenshots = function(){//should probably pass something in..
+    return $http({
+      url:'/api/screenshots/',
+      method:"GET",
+      params: {gruble:uname}
```

Comment:
`params` is incorrect syntax. Also, it's unclear what you're passing in... what is `gruble`? Or `uname`?